aemedge/blocks/related-articles/related-articles.js

```diff
@@ -25,7 +27,10 @@ function getPictureCard(article, placeholders) {
 }
 
 export default async function decorateBlock(block) {
-  const pageTags = getMetadata('article:tag').split(',');
+  const pageTags = getMetadata('article:tag')
+    .split(',')
+    .map((tag) => tag.trim())
+    .map((tag) => toClassName(tag));
```

Comment:
This could be one line? `toClassName(tag.trim())`

Reply:
fixed